Specs/Scene/ExpressionSpec.js

```diff
@@ -783,6 +922,56 @@ defineSuite([
 
         expression = new Expression('color(\'green\') != color(\'green\')');
         expect(expression.evaluate(frameState, undefined)).toEqual(false);
+
+        expression = new Expression('color(\'green\') !== color(\'green\')');
+        expect(expression.evaluate(frameState, undefined)).toEqual(false);
+    });
+
+    it('evaluates vector operations', function() {
+        var expression = new Expression('+vec3(1, 2, 3)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(new Cartesian4(1, 2, 3, 0));
+
+        expression = new Expression('-vec3(1, 2, 3)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(new Cartesian4(-1, -2, -3, 0));
+
+        expression = new Expression('vec2(1, 2) + vec4(3, 4, 5, 6)');
```

Comment:
Here and elsewhere, does GLSL allow this?  I think this should be more strictly typed.